### PR TITLE
add datacite resource types

### PR DESCRIPTION
**Pull request by broughd-qld**
Including the datacite resource types which will be used by DES for their data catalogue.

### CODE REVIEW

File: vocabularies/DataCite-resourceType.ttl [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+@prefix version: <http://purl.org/linked-data/version#> .
+@prefix rr-cat: <http://registry.it.csiro.au/structure/category/> .
+@prefix rr-org: <http://registry.it.csiro.au/structure/org/> .
+@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix ldp:   <http://www.w3.org/ns/ldp#> .
+@prefix bibo:  <http://purl.org/ontology/bibo/> .
+@prefix rr-ui: <http://registry.it.csiro.au/structure/ui/> .
+@prefix time:  <http://www.w3.org/2006/time#> .
+@prefix vs:    <http://www.w3.org/2003/06/sw-vocab-status/ns#> .
+@prefix dc:    <http://purl.org/dc/elements/1.1/> .
```
Comment (KellyVance) [inline vocabularies/DataCite-resourceType.ttl:39]:
```suggestion
@prefix owl:   <http://www.w3.org/2002/07/owl#> .
@prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
@prefix skos:  <http://www.w3.org/2004/02/skos/core#> .
@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
@prefix dct:   <http://purl.org/dc/terms/> .
@prefix reg:   <http://purl.org/linked-data/registry#> .
```

A large number of prefixes not used within the body of the vocabulary. Preferable to remove these, unless there is a known need in the foreseeable future to augment the vocabulary using these additional namespaces.

File: vocabularies/DataCite-resourceType.ttl [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+        skos:scopeNote   "Audio recording. Ex: https://data.datacite.org/application/vnd.datacite.datacite+xml/10.7282/T3J67F05"@en .
+
+<http://registry.it.csiro.au/def/datacite/resourceType/Workflow>
+        a                skos:Concept ;
+        skos:prefLabel   "Workflow"@en ;
+        skos:inScheme    <http://registry.it.csiro.au/def/datacite/resourceType> ;skos:topConceptOf    <http://registry.it.csiro.au/def/datacite/resourceType> ;
+        dct:description  "A structured series of steps which can be executed to produce a final outcome, allowing users a means to specify and enact their  work in a more reproducible manner.\n"@en ;
+        dct:source       <https://schema.datacite.org/meta/kernel-4.1/doc/DataCite-MetadataKernel_v4.1.pdf> ;
+        reg:notation     "Workflow" ;
+        skos:scopeNote   "<p>Computational workflows involving sequential operations made on data by wrapped software and may be specified in a format belonging to a workflow management system, such as Taverna (http://www.taverna.org.uk/).</p><p>An education module on workflows prepared by DataONE is available at http://www.dataone.org/sites/all/documents/L10_AnalysisWorkflows.pptx</p>"@en .
```
Comment (KellyVance) [inline vocabularies/DataCite-resourceType.ttl:59]:
```suggestion
        skos:scopeNote   "Computational workflows involving sequential operations made on data by wrapped software and may be specified in a format belonging to a workflow management system, such as Taverna (http://www.taverna.org.uk/).An education module on workflows prepared by DataONE is available at http://www.dataone.org/sites/all/documents/L10_AnalysisWorkflows.pptx"@en .
```

Suggest removal of HTML 'P' tag

File: vocabularies/DataCite-resourceType.ttl [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+        rdfs:seeAlso     <http://doi.org/10.5438/0014> ;
+        dct:description  "<p>List of resource-types from <a href=\"https://schema.datacite.org/meta/kernel-4.1/include/datacite-resourceType-v4.1.xsd\">DataCite metadata schema</a></p>"@en ;
```
Comment (KellyVance) [inline vocabularies/DataCite-resourceType.ttl:78]:
```suggestion
        dct:description  "List of resource-types from https://schema.datacite.org/meta/kernel-4.1/include/datacite-resourceType-v4.1.xsd DataCite metadata schema"@en ;
```

Suggest removing html tags

**Review by KellyVance — CHANGES_REQUESTED** [review]
Some small improvements to optimise format. But technically sound.

File: vocabularies/DataCite-resourceType.ttl (hunk 1 above)
Comment (broughd-qld) [reply vocabularies/DataCite-resourceType.ttl:39]: Changes accepted.

File: vocabularies/DataCite-resourceType.ttl (hunk 2 above)
Comment (broughd-qld) [reply vocabularies/DataCite-resourceType.ttl]: changes accepted

File: vocabularies/DataCite-resourceType.ttl (hunk 3 above)
Comment (broughd-qld) [reply vocabularies/DataCite-resourceType.ttl:45]: changes accepted.

**Comment by broughd-qld** [comment]
@KellyVance - all changes accepted. Will go through next batch of vocabs and ensure the same issues are resolved.

**Review by KellyVance — APPROVED** [review]
Approved.
Good addition to the catalogue of vocabularies.
Have noticed after-the-fact that concepts have PrefLabels PhysicalObject and DataPaper.
As labels are typically natural language I would personally edit them to have spaces i.e. "Physical Object" and "Data Paper", but I'll leave this to your discretion in case you have particular needs for the labels you use.

**Comment by broughd-qld** [comment]
I have added a space to prefLabel for PhysicalObject, DataPaper and InteractiveResource. No other changes made.

**Review by johnmckellar — APPROVED** [review]

**Comment by broughd-qld** [comment]
@KellyVance This is the Resource Type we discussed yesterday, we will use to distinguish the different types of datasets in our catalogue, potentially has some broader use with your knowledge hub in the future.

File: vocabularies/DataCite-resourceType.ttl [hunk 4]
```diff
@@ -0,0 +1,179 @@
+@prefix owl:   <http://www.w3.org/2002/07/owl#> .
```
Comment (DavidCrosswellGSQ) [inline vocabularies/DataCite-resourceType.ttl:1]:
```suggestion
@prefix rtype: <http://linked.data.gov.au/def/resource-type/> .
@prefix owl:   <http://www.w3.org/2002/07/owl#> .
```

File: vocabularies/DataCite-resourceType.ttl [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+@prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
+@prefix skos:  <http://www.w3.org/2004/02/skos/core#> .
+@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dct:   <http://purl.org/dc/terms/> .
+@prefix reg:   <http://purl.org/linked-data/registry#> .
+
+<http://registry.it.csiro.au/def/datacite/resourceType/Sound>
```
Comment (DavidCrosswellGSQ) [inline vocabularies/DataCite-resourceType.ttl:8]:
```suggestion
<http://linked.data.gov.au/def/resource-type/>
```

**Review by DavidCrosswellGSQ — CHANGES_REQUESTED** [review]
I recommend that we turn this into a linked.data.gov.au vocab. Please update to suit.
We can put skos:exactMatch to the CSIRO concept.
It would be good for CSIRO to move to the linked.data.gov.au.
If DataCite make the concepts available as Linked Data, then we can look to amalgamate.

File: vocabularies/DataCite-resourceType.ttl (hunk 4 above)
Comment (broughd-qld) [reply vocabularies/DataCite-resourceType.ttl:1]: updating to linked.data.gov.au

**Comment by broughd-qld** [comment]
@DavidCrosswellGSQ have updated to linked.data.gov.au as per your request.

**Review by DavidCrosswellGSQ — APPROVED** [review]
I have refactored this vocab to match TTL formats.

**Pull request merged**